### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by nineinchnick**
Joda time and Checkstyle were recently updated and this PR bumps them again to the latest version available.

It also updates Jackson and JMH.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Airbase 129
   - modernizer 2.4.0 (from 2.3.0)
   - joda-time 2.10.14 (from 2.10.13)
   - slf4j 1.7.36 (from 1.7.32)
-  - checkstyle 10.3.2 (from 9.2)
+  - checkstyle 10.3.3 (from 9.2) and maven-checkstyle-plugin 3.2.0 (from 3.1.2)
```
Comment (electrum) [inline CHANGES.md:10]: The Maven plugin should be in a "Plugin updates" section

Comment (electrum) [inline CHANGES.md:10]: The "from" version is wrong, should be 10.3.2

File: CHANGES.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ Airbase 129
 * Dependency updates:
   - spotbugs-annotations 4.7.2 (from 4.3.0)
   - modernizer 2.4.0 (from 2.3.0)
-  - joda-time 2.10.14 (from 2.10.13)
+  - joda-time 2.11.1 (from 2.10.13)
```
Comment (electrum) [inline CHANGES.md:8]: The "from" version should be 2.10.14

File: CHANGES.md (hunk 1 above)
Comment (nineinchnick) [reply CHANGES.md]: Done!

File: CHANGES.md (hunk 2 above)
Comment (nineinchnick) [reply CHANGES.md:8]: This file lists changes vs the previous release and I updated joda-time recently in caa3e0847b47bce9a1a827fee0de83a5fdbe88da, which was not released yet. I tried explaining that in the PR description, but it probably could be made more clear.

File: CHANGES.md (hunk 1 above)
Comment (nineinchnick) [reply CHANGES.md]: It's the same case as with joda-time, it was recently updated and was not released yet.

**Review by nineinchnick — COMMENTED** [review]
I fixed commit messages too.

**Pull request merged**